### ISSUE TITLE
[MAGETWO-1556] Export: Unable to Filter Data by Attribute With Input Type Multiple Select

### ISSUE THREAD

**Issue by piotrekkaminski** [magento-engcom/import-export-improvements#33]
_From @piotrekkaminski on November 2, 2017 20:26_

When trying to filter data by attribute with input type = Multiple Select, no data is exported to csv file.

### Preconditions

There is a custom attribute created with input type Multiple Select, 3 options enabled ("Option 1", "Option 2", "Option 3").
There is a customer in the system created with custom attribute value = "Option 1" and "Option 2".
Version 2.0

### Steps to reproduce

1. Go to System-> Import/Export-> Export at backend
2. Select "Customers" entity type
3. Select "Magento 1.7 format"
4. In the grid for attribute from precondition select "Option 1" filter
5. Press "Continue" button

### Expected Result
The file was exported with customer from precondition

### Actual Result
1. File was not exported. 
2. The error message appears "There is no data for export"

### Note
The same situation with "Magento 2.0" Export format for all customer entity types

_Copied from original issue: magento/magento2#11974_

**Comment by piotrekkaminski** [comment]
_From @magento-engcom-team on November 3, 2017 9:49_

@piotrekkaminski , thank you for your report. Please follow these guidelines for proper tracking of your issue. You can report Commerce-related issues in one of two ways:
You can use the Support portal associated with your account
or
If you are a Partner reporting on behalf of a merchant, use the Partner portal.

GitHub is intended for Magento Open Source users to report on issues related to Open Source only. There are no account management services associated with GitHub.

**_The feature is not available for Magento Open Source._**

**Comment by piotrekkaminski** [comment]
Possibly affects EE only

**Comment by php4umagento** [comment]
I can see two issues - on the export form - multiselect is rendered as a normal select, then when you pick options it is not filtered

**Comment by dmanners** [comment]
Hi @php4umagento during testing the QA team validated the customer section is fixed but would it also be possible to fix the product filter part as well?

```
Steps to reproduce

1) Create product attribute with type multiselect and add to attribute set.
2) Create several product with created attribute.
3) Go to Export and select Products
4) Observe that multiselect attributes are presented as multiselect form field
5) Select multiselect attribute and any option in filter
6) Press "Continue" button
 Expected result
   - Data is filtered by selected option.
 Actual result
   - Exported all products, not looking at the filter.
```

**Comment by dmanners** [comment]
This has been merged into `2.3-develop` as part of https://github.com/magento-engcom/import-export-improvements/pull/98
